src/steamship/data/tags/tag_constants.py

```diff
@@ -161,92 +165,25 @@ class TokenTag(str, Enum):
     LANGUAGE = "language"
 
 
-class SentimentTag(str, Enum):
-    """A set of `name` constants for Tags with a `kind` of `TagKind.sentiment`."""
-
-    POSITIVE = "positive"
-    NEUTRAL = "neutral"
-    NEGATIVE = "negative"
-    SCORE = "score"
-
-
-class EmotionTag(str, Enum):
-    """A set of `name` constants for Tags with a `kind` of `TagKind.emotion`."""
-
-    POSITIVE = "positive"
-    NEUTRAL = "neutral"
-    NEGATIVE = "negative"
-    HAPPINESS = "happiness"
-    SADNESS = "sadness"
-    JOY = "joy"
-    LOVE = "love"
-    ANGER = "anger"
-    FEAR = "fear"
-    SURPRISE = "surprise"
-    HUMOR = "humor"
-    CONCERN = "concern"
-    SERIOUSNESS = "seriousness"
-    SCORE = "score"
-
-
-class IntentTag(str, Enum):
-    """A set of `name` constants for Tags with a `kind` of `TagKind.intent`."""
-
-    SALUTATION = "salutation"
-    PRAISE = "praise"
-    COMPLAINT = "complaint"
-    QUESTION = "question"
-    REQUEST = "request"
-    EXPLANATION = "explanation"
-    SCHEDULING_REQUEST = "scheduling-request"
-    ARE_YOU_THERE = "are-you-there"
-    REVISITING_TOPIC = "revisiting-topic"
-
-
-class TagValue(str, Enum):
+class TagValueKey(str, Enum):
     """A set of key constants for the `value` object within a tag.."""
 
     # Catch-all for confidence, score, ranking
     SCORE = "score"
 
-    # A list of numbers. E.g. for an embedding
-    VECTOR_VALUE = "vector-value"
-    NUMBER_VALUE = "number-value"
-    BOOL_VALUE = "bool-value"
-    STRING_VALUE = "string-value"
+    # A list of numbers
+    VALUE = "value"
```

Comment:
@dkolas what do you think about the `X-value` versus `value` bit here?

I'm relatively agnostic, but I recall you having specific thoughts on this.